siteapp/views.py

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
I'm sorry for being pedantic, but may you remove this empty line?

Reply:
My mistake, was removed.

Reply:
Got me covered, looks good, thanks